src/schema/rules/datatypes/anat.yaml

```diff
@@ -14,9 +14,9 @@ nonparametric:
   - FLASH  # deprecated
   - PD  # deprecated
   extensions:
-  - .nii.gz
-  - .nii
-  - .json
+  - niigz
```

Comment:
can we maybe not do this? =^.^=

see https://github.com/bids-standard/bids-specification/pull/1203#issuecomment-1216762725

Reply:
I mean, I still think we should, but it's up to the maintainers/steering group now.